ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/save-button/save-button.tsx

```diff
@@ -73,6 +73,8 @@ const CheckIcon = styled<SaveButtonProps, 'span'>('span')`
 
 const ModifiedButton = styled(Button)`
   height: 100%;
+  min-height: 0px;
+  line-height: inherit;
```

Comment:
Is this not the default behavior?

Reply:
No
![image](https://user-images.githubusercontent.com/4467636/59702056-be522380-91ab-11e9-956d-9e29bf497263.png)